### ISSUE TITLE
Error when parse hook

### ISSUE THREAD

**Issue by jimmyshiau** [DirectMyFile/github.dart#121]
```
Cause: type 'List<dynamic>' is not a subtype of type 'List<String>' in type cast
#0      Object._as (dart:core/runtime/libobject_patch.dart:67:25)
#1      Hook.fromJSON (file:///Users/jimmy/.pub-cache/hosted/pub.dartlang.org/github-4.0.0/lib/src/common/model/repos_hooks.dart:38:34)
#2      RepositoriesService.createHook.<anonymous closure> (file:///Users/jimmy/.pub-cache/hosted/pub.dartlang.org/github-4.0.0/lib/src/common/repos_service.dart:428:51)
#3      GitHub.postJSON (file:///Users/jimmy/.pub-cache/hosted/pub.dartlang.org/github-4.0.0/lib/src/common/github.dart:283:19)
<asynchronous suspension>
#4      RepositoriesService.createHook (file:///Users/jimmy/.pub-cache/hosted/pub.dartlang.org/github-4.0.0/lib/src/common/repos_service.dart:427:20)
```

**Comment by jimmyshiau** [comment]
Is it shall cast list?
```
 events = input['events'].cast<String>();
```

**Comment by pq** [comment]
I think this is this fixed w/ #122?

**Comment by robbecker-wf** [comment]
Fixed in #122
